test/CMakeLists.txt

```diff
@@ -50,7 +50,8 @@ else ()
   include_directories (BEFORE ../include/lp_oracles)
   include_directories (BEFORE ../include/misc)
 
-
+  #for Eigen
+  add_compile_definitions("EIGEN_NO_DEBUG")
```

Comment:
this needs `cmake` version at least 3.12 Please use conditionally the old syntax `add_compile_options(-D "EIGEN_NO_DEBUG")`

Reply:
is it ok now?